src/ActionList/Item.tsx

```diff
@@ -199,17 +212,41 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
 
     const containerProps = _PrivateItemWrapper ? {role: role || itemRole ? 'none' : undefined} : menuItemProps
 
-    const wrapperProps = _PrivateItemWrapper ? menuItemProps : {}
+    const wrapperProps = _PrivateItemWrapper
+      ? menuItemProps
+      : {
+          sx: {
+            display: 'flex',
+            paddingX: isTopLevelInteractive() ? 0 : 2,
+            paddingY: isTopLevelInteractive() ? 0 : '6px',
```

Comment:
Non-blocking nitpick: we could avoid repeating ourselves if we do this instead

```suggestion
            paddingX: styles.paddingX,
            paddingY: styles.paddingY,
```

If we don't do this, I'd recommend adding a comment about '6px' being a custom value.